utils/install.py

```diff
@@ -295,32 +277,34 @@ def compare(self, version2, separator=". |-", ignorecase=True):
 }
 
 SUPPORTED_EXTENSIONS_VERSION = {
-    "Linux" + "x86_64" + TENSORFLOW: VersionString("0.8.1"),
-    "Linux" + "x86_64" + IMAGE: VersionString("0.8.1"),
-    "Linux" + "x86_64" + IMAGE_DEPS: VersionString("0.8.2"),
-    "Linux" + "amd64" + TENSORFLOW: VersionString("0.8.1"),
-    "Linux" + "amd64" + IMAGE: VersionString("0.8.1"),
-    "Linux" + "arm64" + TENSORFLOW: VersionString("0.8.1"),
-    "Linux" + "arm64" + IMAGE: VersionString("0.8.1"),
-    "Linux" + "armv8" + TENSORFLOW: VersionString("0.8.1"),
-    "Linux" + "armv8" + IMAGE: VersionString("0.8.1"),
+    "Linux" + "x86_64" + TENSORFLOW: VersionString("0.9.0"),
+    "Linux" + "x86_64" + IMAGE: VersionString("0.9.0"),
+    "Linux" + "x86_64" + IMAGE_DEPS: VersionString("0.9.0"),
```

Comment:
Since the version of `0.9.0`, the `wasmedge-image` doesn't need the deps.
You can remove all the `IMAGE_DEPS` related code.

Reply:
Thanks, on it.